app-web/src/store/reducers/siphon.js

```diff
@@ -63,18 +82,59 @@ const findFilterGroup = (state, key) => {
  * sets 'active' property on a filter config object
  * @param {Object} state
  * @param {String} key
+ * @param {Boolean} isActive
  */
-const toggleFilter = (state, key) => {
+const toggleFilter = (state, key, isActive) => {
   const newState = { ...state };
-  const fg = findFilterGroup(state, key);
+  const fg = findFilter(state, key);
+  fg.active = isActive;
 
-  fg.active = !fg.active;
-  const newFilterGroups = newState.filterGroups.map(f => {
+  const newFilters = newState.filters.map(f => {
```

Comment:
could probably be `map(f => f.key === fg.key ? fg : f);`

Reply:
Definitely. Thanks!